source/2_platform/rex_opengl/src/opengl_renderer.cpp

```diff
@@ -20,29 +18,37 @@ namespace rex
   //-------------------------------------------------------------------------
   enum class ClearBits
   {
-    REX_CLEAR_COLOUR_BUFFER = GL_COLOR_BUFFER_BIT,
-    REX_CLEAR_DEPTH_BUFFER = GL_DEPTH_BUFFER_BIT,
-    REX_CLEAR_STENCIL_BUFFER = GL_STENCIL_BUFFER_BIT,
+    RexClearColorBuffer   = GL_COLOR_BUFFER_BIT,
```

Comment:
actually.. why is there a REX prefix here?

Reply:
Because they where defines first, copy paste error